ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -396,7 +396,27 @@ tests:
         not @fips'
       TAG_VERSION: '@4.12'
       TEST_ADDITIONAL: Hypershift
-      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    test:
+    - ref: cucushift-helper-service-build-info
+    - ref: cucushift-pre
+    - ref: openshift-extended-test
+    - ref: cucushift-e2e
+    - ref: openshift-extended-web-tests
+    - ref: cucushift-e2e-report
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: e2e-aws-ipi-ovn-hypershift-guest-p2-destructive
+  cron: 0 6 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and @hypershift-hosted and
+        not @fips'
```

Comment:
Also need `and @destructive`
And the step should be `cucushift-e2e-destructive` (instead of `cucushift-e2e`).